GeosCore/drydep_mod.F90

```diff
@@ -1338,6 +1342,7 @@ SUBROUTINE DEPVEL( Input_Opt, State_Chm, State_Diag, State_Grid, &
     !$OMP PRIVATE( DUMMY1,  DUMMY2,  DUMMY3,  DUMMY4, DAIR,    RB       ) &
     !$OMP PRIVATE( C1X,     VK,      I,       J,      IW                ) &
     !$OMP PRIVATE( DIAM,    DEN,     XLAI_FP, SUNCOS_FP,       CFRAC_FP ) &
+    !$OMP PRIVATE( VTSoutput,    VTSoutput_                             ) &
```

Comment:
Would also recommend setting `VTSoutput = 0.0_fp` and `VTSoutput_ = 0.0_fp` at the top of the parallel DO loop to prevent numerical noise caused by leftover values carrying over from other loop iterations.

Reply:
Will do.